lineman/app/services/thread_query_service.coffee

```diff
@@ -43,7 +43,7 @@ angular.module('loomioApp').factory 'ThreadQueryService', (Records, CurrentUser)
     applyFilters = (view, filters, queryType) ->
       filters = [].concat filters
 
-      view.applyFind(discussionReaderId: { $gt: 0 })
+      view.applyFind(discussionReaderId: { $gt: 0 }) if AbilityService.isLoggedIn()
```

Comment:
Otherwise the logged out user never sees a list of discussions (on the group page for example)